frontend/public/components/image-stream-tag.tsx

```diff
@@ -34,6 +35,90 @@ const splitEnv = (nameValue: string) => {
   };
 };
 
+const supportedPlatformColumnClasses = [
+  'pf-m-hidden pf-m-visible-on-sm',
+  'pf-m-hidden pf-m-visible-on-sm',
+  'pf-m-hidden pf-m-visible-on-lg',
+];
+
+const SupportedPlatformsTableRows = ({ componentProps: { data } }) => {
+  return _.map(data, (submanifest: RowSupportedPlatformData) => {
+    const { os, architecture, digest } = submanifest;
+    return [
+      {
+        title: os,
+        props: {
+          className: supportedPlatformColumnClasses[0],
+        },
+      },
+      {
+        title: architecture,
+        props: {
+          className: supportedPlatformColumnClasses[1],
+        },
+      },
+      {
+        title: digest,
+        props: {
+          className: supportedPlatformColumnClasses[2],
+        },
+      },
+    ];
+  });
+};
+
+export const SupportedPlatformsTable = (props) => {
+  const { t } = useTranslation();
+  const { submanifests, policy, ...tableProps } = props;
+
+  const SupportedPlatformsTableHeader = () => [
+    {
+      title: t('public~OS'),
+      sortField: 'os',
+      transforms: [sortable],
+      props: { className: supportedPlatformColumnClasses[0] },
+    },
+    {
+      title: t('public~Architecture'),
+      sortField: 'architecture',
+      transforms: [sortable],
+      props: { className: supportedPlatformColumnClasses[1] },
+    },
+    {
+      title: t('public~Identifier'),
+      sortField: 'digest',
+      transforms: [sortable],
+      props: { className: supportedPlatformColumnClasses[2] },
+    },
+  ];
+
+  if (!policy || submanifests.length === 0) {
+    // If the policy does not support Manifest Lists, it exits.
+    // or the Manifest List is empty
+    return null;
```

Comment:
if there are no `submanifests` we should return `EmptyBox` component.
Check howe its done for [VolumesTable](https://github.com/openshift/console/blob/master/frontend/public/components/volumes-table.tsx#L213-L232)